icicle/include/icicle/utils/rand_gen.h

```diff
@@ -2,4 +2,16 @@
 #include <random>
 
 inline std::mt19937 rand_generator = std::mt19937{std::random_device{}()};
-static void seed_rand_generator(unsigned seed) { rand_generator.seed(seed); }
\ No newline at end of file
+static void seed_rand_generator(unsigned seed) { rand_generator.seed(seed); }
+
+/**
+ * @brief Generate random unsigned integer in range (inclusive)
+ * @param min Lower limit.
+ * @param max Upper limit.
+ * @return Random (uniform distribution) unsigned integer s.t. min <= integer <= max.
+ */
+static uint32_t rand_int(uint32_t min = 0, uint32_t max = UINT32_MAX)
```

Comment:
change the name to rand_uint_32b
so people will know that the rabge is 32b and not 64